include/deal.II/grid/tria_accessor.templates.h

```diff
@@ -2182,6 +2124,34 @@ TriaAccessor<structdim, dim, spacedim>::face_indices() const
 }
 
 
+
+template <int structdim, int dim, int spacedim>
+inline const internal::ReferenceCellInfo::Base &
+TriaAccessor<structdim, dim, spacedim>::reference_cell_info() const
```

Comment:
We should move this function into `reference_cell_info.h`: #10645 needs this functionality too.

Reply:
I'll work on it in 30 minutes!

Reply:
Let's keep it like this for now. I would like to introduce the option that the right `ReferenceCell::internal::Info` is used based on the `ReferenceCell::Type` in follow-up PR since I consider it a (internal) function change.